c_plus_plus/src/model/classifier/classifier.cpp

```diff
@@ -0,0 +1,30 @@
+#include <cmath>
+
+#include "classifier.h"
+
+namespace Model
+{
+
+std::vector<int> Classifier::classify(std::vector<SourcePosition> audioPositions, std::vector<SphericalAngleRect> imagePositions, int rangeThreshold)
+{
+    std::vector<int> sourcesToSuppress;
+
+    for(size_t i = 0; i < audioPositions.size(); i++)
```

Comment:
I think it would be more readable with a foreach positions in audioPositions. same thing with imagePositions

Reply:
```
for (const auto& audioPosition : audioPositions)
{
    audioPosition.azimuth....
}
```

Reply:
I need to have the index, to be able to tell which sources need to be suppressed. That's why I didn't use a foreach loop.

Reply:
👍 I didn`t see the push_back